git_pile/git_pile.py

```diff
@@ -2024,6 +2024,8 @@ def run(self):
 def get_refs_from_linearized(incremental, pile_branch, start_ref, linear_branch, notes_ref):
     if start_ref:
         pile_range = f"{start_ref}^..{pile_branch}"
+    else:
+        pile_range = pile_branch
```

Comment:
are you passing -r to avoid the incremental mode?  In incremental mode it will try to find the ref to continue from. For that it needs refs/notes/{linear_branch}  to be present in your local repo (if you previously generated in another repo, you can fetch the notes from there.

Reply:
No I am just doing this (this is with tip-fix-genlinear). I don't know why it is getting 2704 commits from, internal-linear is behind by 50 commits. This is the current IGT repo. Thanks.

```
$ git pile genlinear-branch -b internal-linear
‣ Generating branch for 7ae818f4eaf6192f1552a072486726b5c2e95e82 (1/2704)  [ 0.47s ]
Overwriting existing notes for object a5633edd034f333963338fe625ac616bd9ee4d49
‣ Generating branch for 0807973405197b9eb31cf42b95d2d7fed1e1d2f2 (2/2704)  [ 0.16s ]
Overwriting existing notes for object 11c98436e181bdfee0c33ce986c46791fa92427e
‣ Generating branch for 254ebd13389d267b0124cddde72885605d306ada (3/2704)  [ 0.18s ]
Overwriting existing notes for object 0e43ef35015bf1acdd9dff8032f024fdcdb2dfc4
‣ Generating branch for ae75bf60215ce7e3066904d72d7b5464e31f0e84 (4/2704)  [ 0.16s ]
Overwriting existing notes for object 4ef1b750f872a402f36f778b656fd5227d2bd645
‣ Generating branch for 01fac49d24f47df7a87cda1d64c01bb50a3adc68 (5/2704)  [ 0.17s ]
```

Reply:
are you sure you internal-linear branch is pointing to the right ref and not something else due to previous runs with the bug we had?